core/domain/exp_fetchers_test.py

```diff
@@ -252,6 +253,38 @@ def test_retrieval_of_multiple_explorations(self):
             exp_fetchers.get_multiple_explorations_by_id(
                 exp_ids + ['doesnt_exist'])
 
+    def test_get_exploration_user_data(self):
+        auth_id = 'test_id'
+        username = 'testname'
+        user_email = 'test@email.com'
+        user_id = user_services.create_new_user(auth_id, user_email).user_id
+        user_services.set_username(user_id, username)
+        self.assertIsNone(exp_fetchers.get_exploration_user_data(
+            user_id, self.EXP_1_ID))
```

Comment:
Separate this into a separate test function

Reply:
Each unit test function should preferably use only one code path

Reply:
Done!